vector/src/main/java/im/vector/riotx/features/settings/VectorSettingsGeneralFragment.kt

```diff
@@ -858,44 +858,23 @@ class VectorSettingsGeneralFragment : VectorSettingsBaseFragment() {
      * Update the displayname.
      */
     private fun onDisplayNameClick(value: String?) {
-        notImplemented()
-        /* TODO
-        if (!TextUtils.equals(session.myUser.displayname, value)) {
+        value ?: return
```

Comment:
We don't need that "ugly" check, because currentDisplayName cannot be null (2 lines below ?: ""). But we can keep it if getting user from session is an expensive operation.

Reply:
I change that, thanks